angr/procedures/posix/strtok_r.py

```diff
@@ -26,8 +26,8 @@ def run(self, str_ptr, delim_ptr, save_ptr, str_strlen=None, delim_strlen=None):
         strstr = angr.SIM_PROCEDURES["libc"]["strstr"]
         strlen = angr.SIM_PROCEDURES["libc"]["strlen"]
 
-        l.debug("Doin' a strtok_r!")
```

Comment:
I think this one was intentional? If adding a g we should remove the '

Reply:
This one was intentional.

Reply:
undone